airdrop/application/services/airdrop_services.py

```diff
@@ -130,14 +123,73 @@ def get_airdrop_window_stake_details(self, inputs):
 
         return status, response
 
-    def get_stake_info(self, staking_contract_address, address):
-        contract = get_contract_instance(
-            STAKING_CONTRACT_PATH, staking_contract_address, contract_name='STAKING')
+    def get_stake_info(self, contract_address, address, airdrop_rewards):
```

Comment:
can we call address as user_address here ?

Reply:
Updated the variable to `user_wallet_address` in [commit](https://github.com/singnet/airdrop-services/pull/94/commits/387f89de55b5762ca4b37cfdc8aaef360afee03e)